### PR TITLE
Conservativity of freely adding 1,× to a free Category

### PR DESCRIPTION
**Pull request by hejohns**
WTS that over a Quiver Q, the inclusion of the free category over Q into the free cartesian category over Q (as a trivial ProductQuiver) is fully faithful.

Faithfulness is quick, but we're going to use a gluing normalization argument (over the free cartesian category) for fullness.

WIP: a very concrete displayed category of presheaves, given directly, eg not as slices, not as a displayed category of displayed functors and natural transformations

### CODE REVIEW

**Comment by maxsnew** [comment]
Given a presheaf `P` on `C`, the slick definition of a dependent presheaf over `P` is a presheaf on the category of elements of `P`: https://ncatlab.org/nlab/show/slice+of+presheaves+is+presheaves+on+slice

**Comment by hejohns** [comment]
This could be merged after #102, and a decent amount of cleanup.

File: Cubical/Categories/Displayed/Instances/Presheaf/Base.agda [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+private
+  variable ℓC ℓC' ℓD ℓD' ℓE ℓE' : Level
+
+module _ (C : Category ℓC ℓC') (ℓSET ℓSETᴰ : Level) where
+  module _ (P : Presheaf C ℓSET) where
+    -- TODO: this is a name clash with the other Presheafᴰ, which is different
+    -- (and not a displayed presheaf in the strictest sense of "displayed X")
+    Presheafᴰ : Type _
+    Presheafᴰ = Presheaf (∫ᴾ P) ℓSETᴰ
```
Comment (hejohns) [inline Cubical/Categories/Displayed/Instances/Presheaf/Base.agda:33]: + we need to figure out what to name this notion of displayed presheaf, which is an instance of the already-existing one, although I'm not sure what the definitional differences are, if any.

Comment (maxsnew) [reply Cubical/Categories/Displayed/Instances/Presheaf/Base.agda:33]: Hm good question. This one is a "dependent" presheaf maybe? Would it be so bad to still call this `Presheafᴰ` ? I'm not sure

**Comment by maxsnew** [comment]
@hejohns #102 has been merged now so you can rebase and move forward with this one

File: Cubical/Categories/Displayed/Instances/Presheaf/Base.agda (hunk 1 above)
Comment (hejohns) [reply Cubical/Categories/Displayed/Instances/Presheaf/Base.agda:33]: I've left the name as-is. At least I'm fine with it.

**Comment by hejohns** [comment]
> @hejohns #102 has been merged now so you can rebase and move forward with this one

should be good to go

**Comment by maxsnew** [comment]
Looks great! Excellent work

**Comment by hejohns** [comment]
Thanks, I appreciate it

**Pull request merged**